src/css/ttyg/chat-item-details.css

```diff
@@ -156,3 +156,33 @@
     -webkit-box-orient: vertical;
     max-height: 4.5rem; /* fallback if line-clamp doesn't work */
 }
+
+.chat-detail .token-usage-info-btn {
+    margin-top: 2px;
+}
+
+/*********************************************************
+* Styling the popover displaying token usage information.
+**********************************************************/
```

Comment:
I'm not sure of the styling of the popover content matches the mockup. Can you check that please. For example, the heading `Tokens usage` shoupd probably bigger. Now you are using a div styled with fot-size:1em, but why it's not just h3 or h4?!

Reply:
done